addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ZestStatementFromJson.java

```diff
@@ -52,81 +55,59 @@ public class ZestStatementFromJson {
-                                            json.getInt(FRAME_INDEX),
-                                            json.getString(FRAME_NAME),
-                                            json.getBoolean(FRAME_ISPARENT));
-                    break;
-                case ZEST_CLIENT_WINDOW_RESIZE:
-                    stmt =
-                            (ZestStatement)
-                                    new ZestClientWindowResize(
-                                            json.getString(WINDOW_HANDLE),
-                                            json.getInt(X_VALUE),
-                                            json.getInt(Y_VALUE));
-                    break;
-                default:
-                    throw new Exception("Element Type Not found");
-            }
+            return switch (elementType) {
+                case ZEST_CLIENT_LAUNCH -> new ZestClientLaunch(
+                        json.getString(WINDOW_HANDLE),
+                        json.getString(BROWSER_TYPE),
+                        json.getString(URL),
+                        json.getString(CAPABILITIES),
+                        json.getBoolean(HEADLESS));
+
+                case ZEST_CLIENT_ELEMENT_CLICK -> new ZestClientElementClick(
+                        json.getString(WINDOW_HANDLE),
+                        json.getString(TYPE),
+                        json.getString(ELEMENT));
+                case ZEST_CLIENT_ELEMENT_SEND_KEYS -> new ZestClientElementSendKeys(
+                        json.getString(WINDOW_HANDLE),
+                        json.getString(TYPE),
+                        json.getString(ELEMENT),
+                        json.getString(VALUE));
+                case ZEST_CLIENT_ELEMENT_SUBMIT -> new ZestClientElementSubmit(
+                        json.getString(WINDOW_HANDLE),
+                        json.getString(TYPE),
+                        json.getString(ELEMENT));
+                case ZEST_CLIENT_ELEMENT_CLEAR -> new ZestClientElementClear(
+                        json.getString(WINDOW_HANDLE),
+                        json.getString(TYPE),
+                        json.getString(ELEMENT));
+                case ZEST_CLIENT_WINDOW_CLOSE -> new ZestClientWindowClose(
+                        json.getString(WINDOW_HANDLE), json.getInt(SLEEP_IN_SECONDS));
+                case ZEST_CLIENT_SWITCH_TO_FRAME -> new ZestClientSwitchToFrame(
+                        json.getString(WINDOW_HANDLE),
+                        json.getInt(FRAME_INDEX),
+                        json.getString(FRAME_NAME),
+                        json.getBoolean(FRAME_ISPARENT));
+                case ZEST_CLIENT_WINDOW_RESIZE -> new ZestClientWindowResize(
+                        json.getString(WINDOW_HANDLE), json.getInt(X_VALUE), json.getInt(Y_VALUE));
+                default -> {
+                    Exception e = new Exception("Element type not found " + elementType);
+                    LOGGER.error(e.getMessage(), e);
```

Comment:
IMO we should just throw and log in the existing try/catches as error instead of debug.